eessi_bot_event_handler.py

```diff
@@ -113,7 +114,25 @@ def handle_issue_comment_event(self, event_info, log_file=None):
         # currently, only commands in new comments are supported
         #  - commands have the syntax 'bot: COMMAND [ARGS*]'
 
-        # first check if sender is authorized to send any command
+        # only scan for commands in newly created comments
+        if action == 'created':
+            comment_received = request_body['comment']['body']
+            self.log(f"comment action '{action}' is handled")
+        else:
+            # NOTE we do not respond to an updated PR comment with yet another
+            #      new PR comment, because it would make the bot very noisy or
+            #      worse could result in letting the bot enter an endless loop
+            self.log(f"comment action '{action}' not handled")
+            return
+        # at this point we know that we are handling a new comment
+
+        # check if comment does not contain a bot command
+        if not contains_any_bot_command(comment_received):
+            self.log(f"comment does not contain a bot comment; not processing it further")
```

Comment:
No need for an f string here, a plain string will do.
Can't claim credit, the CI clued me in for this one 😉